__tests__/server.test.js

```diff
@@ -137,6 +137,89 @@ describe("/api/articles/:article_id/comments", () => {
             })
         })
     })
+    describe("POST", () => {
+        test("201: Adds a comment to an article", () => {
+            return request(app)
+            .post("/api/articles/1/comments")
+            .send({
+                username: "butter_bridge",
+                body: "My comment"
+            })
+            .expect(201)
+            .then((response) => {
+                expect(typeof response.body.comment.comment_id).toBe("number")
+                expect(typeof response.body.comment.votes).toBe("number")
+                expect(typeof response.body.comment.created_at).toBe("string")
+                expect(response.body.comment.author).toBe("butter_bridge")
+                expect(response.body.comment.body).toBe("My comment")
+                expect(response.body.comment.article_id).toBe(1)
+            })
+        })
+        test("201: If any extra properties exist on comment object being sent, ignore other properties and add comment as usual", () => {
+            return request(app)
+            .post("/api/articles/1/comments")
+            .send({
+                username: "butter_bridge",
+                body: "My comment",
+                extraKey: "extra content"
+            })
+            .expect(201)
+            .then((response) => {
+                expect(typeof response.body.comment.comment_id).toBe("number")
+                expect(typeof response.body.comment.votes).toBe("number")
+                expect(typeof response.body.comment.created_at).toBe("string")
+                expect(response.body.comment.author).toBe("butter_bridge")
+                expect(response.body.comment.body).toBe("My comment")
+                expect(response.body.comment.article_id).toBe(1)
```

Comment:
how do you know that the extra key is not on the comment returned if you aren't testing this.